packages/mint-components/src/components/tax-and-cash/sqm-tax-document-submitted/sqm-tax-document-submitted.feature

```diff
@@ -0,0 +1,77 @@
+    Given a user is on the Tax Document Submitted view
+
+  @motivating
+  Scenario: View Tax Document Submitted
+    Given the user has submitted a tax document
+    When they view Document Submitted
+    Then they see the status of their Tax Document and Banking Information
+
+  @motivating
+  Scenario: View Tax Document Submitted is loading
+    Given the user has submitted a tax document
+    When they view Document Submitted
+    Then they see a skeleton loaders until the end users Tax Document/Banking information data is available
+
+  @minutia @ui
+  Scenario Outline: The Tax Form header is displayed to the end users
+    Given the user has submitted a W9/W8-BEN/W8-BEN-E Tax form
+    Then the Tax Form header displays "<documentType> Tax Form"
+
+    Examples: 
+      | documentType |
+      | W9           |
+      | W8-BEN       |
+      | W8-BEN-E     |
+
+  @minutia @ui
+  Scenario Outline: The date submitted and status of a Tax Form is displayed to the user
+    Given the Tax Document is currently <status>
+    When they view the Tax Document Submitted
+    Then they see a badge with <status> text and a respective <badgeVariant>
+    Then they see a message indicating "<taxStatusMessage> on <dateSubmitted>"
+
+    Examples: 
+      | status       | badgeVariant | taxStatusMessage                                                          | dateSubmitted |
+      | ACTIVE       | success      | Submitted on                                                              | Jan 17, 2024  |
+      | NOT_VERIFIED | neutral      | Awaiting Review. Submitted on                                             | Jan 17, 2024  |
+      | NOT_ACTIVE   | danger       | Ensure your information matches your profile and resubmit a new document. | Jan 17, 2024  |
+      | EXPIRED      | danger       | Expired on                                                                | Jan 17, 2024  |
+
+  @minutia @ui
+  Scenario: Status badge and text does not appear if participant is not required to submit tax form
+    When a tax document is not required
+    Then tax document header does not include a document type
+    And the description text under the header displays "Tax documents are only required if you are based in the US. If your country of residence has changed, please contact Support"
+    And the "Submit new document" is not available
+
+  @minutia @ui
+  Scenario Outline: A Danger Alert is displayed if the users tax form is invalid
+    Given the <status> is "NOT_ACTIVE" or "EXPIRED"
+    Then a danger alert indicating the <documentType> with a <taxAlertHeader> and <taxAlertMessage> appears
+
+    Examples: 
+      | status     | documentType | taxAlertHeader                                                             | taxAlertMessage                      |
+      | NOT_ACTIVE | W9           | Your W9 tax form has personal information that doesn't match your profile. | Please resubmit a new W9 form.       |
```

Comment:
This not active state can happen if the W8 fails validation. We should not be hardcoding this copy for W9